openpype/hosts/fusion/plugins/publish/render_local.py

```diff
@@ -67,7 +58,23 @@ def process(self, instance):
         repre_preview["tags"] = ["review", "preview", "ftrackreview", "delete"]
```

Comment:
```suggestion
        repre_preview["tags"] = ["review",  "ftrackreview", "delete"]
```

This can also go. As far as I know `preview` tag is already gone for some time. Thanks.